deploy/utils/backup_script/run_backup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
```

Comment:
Why no `set -e`? Any error should cause the run to fail.  It's quite dangerous to continue messing with backups when some error occurs.